### DISCUSSION TITLE
Backup Status?

### DISCUSSION THREAD

**Discussion by Deepintheeast** [dougiteixeira/proxmoxve#342] [Feature request] [answered]
Hi, there 
is it possible to show the backup status?

Greetings, Jens

**Comment by mZ738** [comment, marked as answer]
Backups are covered since 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)): per node `Last backup`, `Backup duration`, `Backup status` (a problem sensor, on when the last run was anything but OK, including partial `job errors`) from the task log, and `Backup running` while a `vzdump` run is in progress — the sensor an automation waits for before shutting a node down. To start one there is the `proxmoxve.backup` action (target the guest, node or cluster devices; storage, mode, compression, notes; one run per node), `Back up now` / `Back up all` buttons behind a storage picked in the options, and a blueprint for a schedule. Details: [Actions](https://github.com/dougiteixeira/proxmoxve/blob/main/docs/actions.md), [Entities](https://github.com/dougiteixeira/proxmoxve/blob/main/docs/entities.md#last-backup-per-node), [Blueprints](https://github.com/dougiteixeira/proxmoxve/blob/main/blueprints/readme.md). Closing as done.
